tests/model_serving/model_runtime/triton/basic_model_deployment/utils.py

```
@@ -21,128 +8,45 @@
-    PROTO_FILE_PATH,
     TEMPLATE_MAP,
-    TRITON_GRPC_PORT,
     TRITON_REST_PORT,
 )
-from utilities.constants import KServeDeploymentType, Labels, Protocols, RuntimeTemplates
+from utilities.constants import KServeDeploymentType, Labels, RuntimeTemplates, Timeout
 
 
 def send_rest_request(url: str, input_data: dict[str, Any]) -> Any:
-    response = requests.post(url=url, json=input_data, verify=False, timeout=180)
+    response = requests.post(url=url, json=input_data, verify=False, timeout=Timeout.TIMEOUT_3MIN)
```

Ruff (0.15.18)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)